src/hera/workflows/runner.py

```diff
@@ -76,6 +84,20 @@ def _is_str_kwarg_of(key: str, f: Callable):
         return False
 
 
+def _map_keys(function: Callable, kwargs: dict):
+    """Change the kwargs's keys to use the python name instead of the parameter name which could be kebab case."""
+    if os.environ.get("script_annotations", None) is not None:
```

Comment:
Just to be clear it's Hera's annotation and avoid naming collisions

```suggestion
    if os.environ.get("hera__script_annotations", None) is not None:
```